app/_components/SideNavigationBar.tsx

```diff
@@ -45,13 +41,13 @@ function SideNavigationBar({
     }
   };
 
-  const handleSelectCategory = (name: typeof selectCategory) => {
+  const handleSelectCategory = (name: typeof selectProductCategory) => {
     handleDrawerClose();
-    setSelectCategory(name);
+    setSelectProductCategory(name);
   };
-  const handleSelectTag = (name: typeof selectTag) => {
+  const handleSelectTag = (name: typeof selectProductTag) => {
```

Comment:
typeof 쓰신 이유가 있을까요? 얼핏 보기에는 string으로 선언 하는거랑 같아 보입니다
```suggestion
  const handleSelectTag = (name: string) => {
```

Reply:
string으로 선언하면 다른 문자열도 가능해서 타입을 명확하게 하기 위해서 typeof로 수정했는데, 이 프로젝트에서는 string으로 해도 될  것 같아서 수정하겠습니다!